### ISSUE TITLE
Ability to change bar graph x axis to field other than time

### ISSUE THREAD

**Issue by jaxxstorm** [Graylog2/graylog2-web-interface#511]
It would be great if it was possible to change the x axis of bar graphs to something other than time, for example if you could change it to reflect a field like "host" (to graph the number of messages per host, for example) or "source" (number of messages per source)

**Comment by vitosamson** [comment]
Is this a planned feature, or has the feature been implemented since this issue was opened?

**Comment by joschi** [comment]
@vitosamson No, it has not been implemented yet and it's not scheduled for a specific milestone right now.

**Comment by Veejan** [comment]
Has this been changed yet in the newer versions ?

**Comment by theabraxas** [comment]
This is something I would also definitely like to see implemented. Being able to make more meaningful graphs/widgets for dashboards would mean a ton.

**Comment by RichNahra** [comment]
+1  I would like to generate a chart on X=host and Y=http status codes within a given time period.

**Comment by xoroz** [comment]
any news? I need to have 2 fileds on the dashboard..

**Comment by eduardm** [comment]
Anyone working on this? I plan to start writing a plugin for this, but I would gladly continue any existing work, if exists.
